pkg/api/handler/default.go

```diff
@@ -388,12 +390,10 @@ func (m ArcDefaultHandler) processTransactions(ctx context.Context, transactions
 	txIds := make([]string, 0)
 	txErrors := make([]interface{}, 0)
 
-	txValidator := defaultValidator.New(m.NodePolicy)
-
 	for len(transactionsHexes) != 0 {
-		isBeefFormat := txValidator.IsBeef(transactionsHexes)
+		hexFormat := validator.GetHexFormat(transactionsHexes)
 
-		if isBeefFormat {
+		if hexFormat == validator.Beef {
```

Comment:
Maybe we should change `validator.Beef` to `validator.FormatBeef` or `validator.BeefFormat`, to avoid confusion

Reply:
`BEEF` and `EF` are acronyms where F stands for `Format`, and I feel a bit anxious when I see names like `BeefFormat`
I understand that the simple `Beef` here can be misleading since we have a package with the same name and a `BeefData` structure. 
In this context, it's about hex formats, so what do you think about names like: `RawHex`, `EfHex`, `BeefHex`?

Reply:
Okay, let's make this compromise and have `RawHex`, `EfHex` and `BeefHex` :)

Reply:
Enum names changed